pkg/agent/core/ngt/service/vqueue/queue.go

```diff
@@ -48,19 +45,17 @@ type Queue interface {
 }
 
 type vqueue struct {
-	ich              chan index // ich is insert channel
-	uii              []index    // uii is un inserted index
-	imu              sync.Mutex // insert mutex
-	uiim             uiim       // uiim is un inserted index map (this value is used for GetVector operation to return queued vector cache data)
-	dch              chan key   // dch is delete channel
-	udk              []key      // udk is un deleted key
-	dmu              sync.Mutex // delete mutex
-	udim             udim       // udim is un deleted index map (this value is used for Exists operation to return cache data existence)
-	eg               errgroup.Group
-	finalizingInsert atomic.Value
-	finalizingDelete atomic.Value
-	closed           atomic.Value
+	ich  chan index // ich is insert channel
+	uii  []index    // uii is un inserted index
+	imu  sync.Mutex // insert mutex
+	uiim uiim       // uiim is un inserted index map (this value is used for GetVector operation to return queued vector cache data)
+	dch  chan key   // dch is delete channel
```

Comment:
we can remove this variable

Reply:
https://github.com/vdaas/vald/pull/1400#discussion_r694552817